routes/users.js

```diff
@@ -265,4 +265,6 @@ router.get('/:username', users.getUser)
 // upload.single('profile') -> multer inmemory storage of file for type multipart/form-data
 router.post('/picture', authenticate, upload.single('profile'), users.postUserPicture)
 
+router.patch('/chainCode', authenticate, users.updateChainCode)
```

Comment:
What's the reason of going ahead with patch request here?

Reply:
We are updating the chain code. There might already be an existing chain code in the user object so we can't use POST here.

Reply:
The endpoint is used for generating a new chain code for that user, regardless of if the person have a pre existing chain code or not.

Reply:
`The endpoint is used for generating a new chain code for that user` this is bit contradicting the above statement here